packages/@aws-cdk/aws-redshift-alpha/lib/cluster.ts

```diff
@@ -558,6 +565,17 @@ export class Cluster extends ClusterBase {
       );
     }
 
+    const nodeType = props.nodeType || NodeType.DC2_LARGE;
+
+    if (props.multiAz) {
+      if (!nodeType.startsWith('ra3')) {
+        throw new Error(`Multi-AZ cluster is only supported for RA3 node types. got: ${nodeType}`);
```

Comment:
for the cases that users do not specify the props.

```suggestion
        throw new Error(`Multi-AZ cluster is only supported for RA3 node types, got: ${props.nodeType}`);
```